cmd/root.go

```diff
@@ -135,11 +137,32 @@ func initRootOpts() error {
 	rootOpts.since = viper.GetString("since")
 	rootOpts.until = viper.GetString("until")
 	rootOpts.title = viper.GetString("title")
-	rootOpts.tokenPath = viper.GetString("token-path")
+
+	token, err := initToken()
+	if err != nil {
+		return err
+	}
+	rootOpts.token = token
 
 	return nil
 }
 
+// initToken parses the github token either from the PULLSHEET_TOKEN env var or
+// from the file given by either PULLSHEET_TOKEN-PATH env var or --token-path
```

Comment:
Any chance we could make this just name GITHUB_TOKEN_PATH? 

PULLSHEET_TOKEN-PATH is especially awkward, with the mismatched delimiters.